tests/gem5/arm_boot_tests/test_linux_boot.py

```diff
@@ -42,6 +42,7 @@ def test_boot(
     memory_class: str,
     length: str,
     to_tick: Optional[int] = None,
+    systemd: bool = False,
```

Comment:
I hate to ask, but... can you make this a required parameter so it is explicit in our tests. I think that improves readability.